spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/convert/DefaultJdbcTypeFactory.java

```diff
@@ -44,25 +44,6 @@ public DefaultJdbcTypeFactory(JdbcOperations operations) {
 		this(operations, org.springframework.data.jdbc.core.dialect.JdbcArrayColumns.DefaultSupport.INSTANCE);
 	}
 
-	/**
```

Comment:
Maybe we can move this in the separate commit? We have a chance to remove it in spring-data-jdbc 4.0 major release